l3kernel/l3fp-symbolic.dtx

```diff
@@ -445,23 +402,23 @@
   }
 \cs_new:Npn \@@_symbolic_unary_to_tl:NNw #1#2 , #3 @
   {
-    \use:ff
+    \use:ee
       { \@@_symbolic_cs_arg_to_fn:NN #1#2 }
       { \exp_last_unbraced:Nf ( { \@@_to_tl_dispatch:w #3 } ) }
   }
 \cs_new:Npn \@@_symbolic_binary_to_tl:Nww #1, #2; #3; @
   {
-    \use:ff
+    \use:ee
       { \exp_last_unbraced:Nf ( { \@@_to_tl_dispatch:w #2; } ) }
       {
-        \use:ff
+        \use:ee
           { \@@_types_cs_to_op:N #1 }
           { \exp_last_unbraced:Nf ( { \@@_to_tl_dispatch:w #3; } ) }
       }
   }
 \cs_new:Npn \@@_symbolic_function_to_tl:Nw #1, #2@
   {
-    \use:ff { \@@_types_cs_to_op:N #1 }
+    \use:ee { \@@_types_cs_to_op:N #1 }
```

Comment:
Again, unnecessary `\use:ee`, also, I haven't checked whether `\@@_symbolic_function_to_tl:Nw` is used outside an `e`-expansion context anywhere, but if not, the `\use:e` can be dropped completely here.